spanner_orm/model.py

```diff
@@ -206,13 +206,25 @@ def create(cls, transaction=None, **kwargs):
   def create_or_update(cls, transaction=None, **kwargs):
     cls._execute_write(api.SpannerApi.upsert, transaction, [kwargs])
 
+  def delete_batch(cls, transaction, models):
+    key_list = []
```

Comment:
Can you document how this and `save_batch` behaves when transaction is None. Because this one seems different than delete. `save_batch` could potentially run 3 different transactions if tx=None, delete will always run in 1 transaction.

I pass the tx as None whenever I can, but I end up needing to look at source code to see if it will only create 1 transaction